src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2

```diff
@@ -279,9 +275,8 @@ All remaining pixels (land, islands in large lakes, small inland waterbodies, an
 a value of 1. The water mask is stored as an 8-bit unsigned integer GeoTIFF file and is always included in the product
 package.
 
-Users can choose to apply the water mask to output products, which affects the wrapped interferogram,
-the unwrapped interferogram, and the browse image. It returns NoData values over the areas covered by the water mask
-in these output images. This product {{ "has" if apply_water_mask else "has not" }} had the water mask applied.
+Users can choose to apply the water mask before phase unwrapping. This product {{ "has" if apply_water_mask else "has not" }} 
+had the water mask applied.
```

Comment:
Is the part about `NoData values` no longer important?

Reply:
I don't believe so.